api/reporting/queries/live/latest_table_views/latest_cohorts.sql

```diff
@@ -0,0 +1,13 @@
+-- All cohorts from the most recent snapshot
+SELECT
+    c.*
+FROM
+    reporting_test.cohort c
+WHERE
+        c.snapshot_timestamp = (
+        SELECT
+            MAX(u2.snapshot_timestamp)
+        FROM
+            reporting_test.user u2)
```

Comment:
Is there a reason to refer to the user table here instead of the cohort table for pulling the latest snapshot timestamp? It might be clearer to keep the subquery within the same table.

Reply:
Good point. I was going to make a view for just the latest timestamp or another one for all the unique timestamps values. I think I have that in another environment.

In the extreme case, everyone could delete all their cohorts today, and on Monday there should be zero cohorts at the latest timestamp, so we need a global notion of what that is.